blog/2024-04-30-opentofu-1-7-0.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "OpenTofu 1.7.0 is out with State Encryption, Dynamic Provider-Defined Functions, and more" 
+slug: opentofu-1-7-0
+description: OpenTofu 1.7.0 is now available with full state encryption, dynamic provider-defined functions, the removed and loopable import blocks, new migration guides, and much more.
+image: /img/blog/opentofu-1-7-0.png
+---
+
+In the last few months [since our first stable release](/blog/opentofu-is-going-ga/) the OpenTofu community and the core team have worked hand in hand to bring functionality to OpenTofu that has been requested for years.  We are proud to announce the immediate availability of [OpenTofu 1.7.0](https://github.com/opentofu/opentofu/releases/tag/v1.7.0), bringing you the following highlights:
+
+- [**Full state encryption**](/docs/intro/whats-new/#state-encryption) protects your state file from prying eyes, no matter what storage backend you use. You can provide your encryption keys securely using environment variables, or using a key management system such as AWS KMS, GCP KMS, or OpenBao. *This feature has been developed in collaboration with Stephan Bartels (Interhyp) and Alex Scheel from the OpenTofu community, whom we would like to thank for their work on this feature.*
```

Comment:
Also
> You can provide your encryption keys securely using environment 

should be

>  You can provide your encryption password securely using environment 

because we push it through pbkdf2, which is a password key derivation function. I know it's potato potato but afaik it's an important distinction for some.